library/build.gradle

```diff
@@ -35,6 +36,13 @@ android {
             java.srcDirs = ['tests/src']
         }
     }
+
+    // This enables long timeouts required on slow environments, e.g. Travis
+    adbOptions {
```

Comment:
FWIW, I don't think this is actually needed, as I believe Travis & Gradle pick up the ABD timeout from the `ADB_INSTALL_TIMEOUT` environmental variable defined in `.travis.yml`.  So bumping Gradle plugin alone may be enough to prevent timeouts.  I don't think it hurts to have both, although I'm not sure which value wins, this timeout value or the `ADB_INSTALL_TIMEOUT`.

Reply:
I checked with sample project. In the above case the timeout value mentioned in gradle wins over ADB_INSTALL_TIMEOUT.